engine/graphics/src/webgpu/graphics_webgpu.cpp

```diff
@@ -800,76 +832,82 @@ void WebGPUContext::instanceRequestAdapterCallback(WGPURequestAdapterStatus stat
     }
 }
 
-bool WebGPUContext::initialize(const ContextParams& params)
+static bool initializeWebGPUContext(WebGPUContext* context, const ContextParams& params)
 {
     TRACE_CALL;
+    context->m_Window = params.m_Window;
+    assert(dmPlatform::GetWindowStateParam(context->m_Window, dmPlatform::WINDOW_STATE_OPENED));
+    context->m_OriginalWidth   = params.m_Width;
+    context->m_OriginalHeight  = params.m_Height;
+    context->m_PrintDeviceInfo = params.m_PrintDeviceInfo;
+    context->m_CurrentUniforms.m_Allocs.SetCapacity(32);
```

Comment:
Set capacity to 32. No background under that value. Simply 32

Reply:
I mean that 32 was selected without any reasoning. Just 32 as the initial capacity value, not "32 because browsers supported maximum 64 uniforms", for example.